definitions/tools/filter_cle_variants.cwl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env cwl-runner
+ 
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Adds an INFO tag (CLE) to variants in the target file present in a CLE-validated vcf file"
+
+requirements:
+    - class: ShellCommandRequirement
+    - class: InlineJavascriptRequirement
+    - class: DockerRequirement
+      dockerPull: "mgibio/bcftools-cwl:1.9"
+    - class: ResourceRequirement
+      ramMin: 8000
+
+baseCommand: ["/opt/bcftools/bin/bcftools", "annotate"]
+arguments:
+    [ "-Oz", "-o", "cle_variants_flagged.vcf.gz" ]  
+
+inputs:
+    cle_variants:
+        type: File?
+        secondaryFiles: [.tbi]
+        inputBinding:
+            position: 1
+            valueFrom: |
+                ${
+                    return [ '-a', inputs.cle_variants.path, '-m', 'CLE' ];
+                }
+    vcf:
+        type: File
+        secondaryFiles: [.tbi]
+        inputBinding:
+            position: 2
+outputs:
+    cle_flagged:
+        type: File
+        outputBinding:
+            glob: "cle_variants_flagged.vcf.gz"
```

Comment:
Do we care about being able to customize this name or base it on the input name?

Reply:
Updated this to be more general in the latest commit. Since this output isn't exposed anywhere except to a downstream tool that immediately renames it, I think it's ok as-is.

Reply:
Definitely works!  In filters I'm currently biased towards basing the output name on the input name by default, just because that's a great way to disambiguate if the same step has to run multiple times in a workflow (especially if both end up being outputs).  None of that currently applies in this case, though! :smile: